app/assets/stylesheets/ghetto_styles.css.scss

```diff
@@ -0,0 +1,1500 @@
+#landing #sign_up form input[type='submit'] {
+  -webkit-transition: background 0.1s linear;
+  -moz-transition: background 0.1s linear;
+  -o-transition: background 0.1s linear;
+  transition: background 0.1s linear;
+  -webkit-box-shadow: 0 0 0 0;
+  -moz-box-shadow: 0 0 0 0;
+  box-shadow: 0 0 0 0;
+  font-size: 1.2em;
+  background: #99CC00;
+  color: #fff;
+  text-shadow: 0 -1px 0 #669900;
+  border: 1px solid #999999;
+}
+/* line 36, ../../app/assets/stylesheets/new_styles/_landing.scss */
+#landing #sign_up form input[type='submit']:hover {
+  background: #91bd0f;
+}
+/* line 40, ../../app/assets/stylesheets/new_styles/_landing.scss */
+#landing #sign_up form input[type='submit']:active {
+  -webkit-box-shadow: 0 0 0 0;
+  -moz-box-shadow: 0 0 0 0;
+  box-shadow: 0 0 0 0;
+  background: #ff442c;
+}
+/* line 48, ../../app/assets/stylesheets/new_styles/_landing.scss */
+#landing #invites {
+  margin: 20px 0;
+}
+/* line 52, ../../app/assets/stylesheets/new_styles/_landing.scss */
+#landing #footer {
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  width: 100%;
+  padding: 10px 0;
+  color: #333;
+  text-align: center;
+}
+/* line 62, ../../app/assets/stylesheets/new_styles/_landing.scss */
+#landing #landing_banner {
+  text-align: center;
+  padding: 50px;
+  margin-bottom: 50px;
+  color: #ccc;
+  background: #222;
+}
+/* line 63, ../../app/assets/stylesheets/new_styles/_landing.scss */
+#landing #landing_banner .container {
+  position: relative;
+}
+/* line 73, ../../app/assets/stylesheets/new_styles/_landing.scss */
+#landing #landing_banner h2 {
+  font-weight: 100;
+}
+/* line 77, ../../app/assets/stylesheets/new_styles/_landing.scss */
+#landing #landing_banner .landing-logo {
+  margin-bottom: 15px;
+  height: 60px;
+}
+/* line 82, ../../app/assets/stylesheets/new_styles/_landing.scss */
+#landing #landing_banner .landing-banner-right {
+  position: absolute;
+  right: 0;
+}
+/* line 86, ../../app/assets/stylesheets/new_styles/_landing.scss */
+#landing #landing_banner .landing-banner-right a {
+  color: #ccc;
+  font-size: 1.2em;
+  font-weight: 100;
+  padding: 5px 10px;
+}
+/* line 94, ../../app/assets/stylesheets/new_styles/_landing.scss */
+#landing #landing_banner .login-button {
+  -moz-border-radius: 3px 3px 3px 3px;
+  -webkit-border-radius: 3px 3px 3px 3px;
+  border-radius: 3px 3px 3px 3px;
+  border: 1px solid #666666;
+  background: #111;
+}
+/* line 99, ../../app/assets/stylesheets/new_styles/_landing.scss */
+#landing #landing_banner .login-button:hover {
+  background: #222;
+  border: 1px solid #999999;
+  text-decoration: none;
+}
+/* line 107, ../../app/assets/stylesheets/new_styles/_landing.scss */
+#landing #steps {
+  text-align: center;
+}
+/* line 109, ../../app/assets/stylesheets/new_styles/_landing.scss */
+#landing #steps p {
+  color: #666;
+}
+
+/* autocomplete colors */
+/* line 2, ../../app/assets/stylesheets/new_styles/_forms.scss */
+input:-webkit-autofill {
+  background-color: white !important;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>!important should not be used